src/datanode/src/region_server.rs

```diff
@@ -1218,7 +1218,12 @@ mod tests {
         );
 
         let response = mock_region_server
-            .handle_request(region_id, RegionRequest::Drop(RegionDropRequest {}))
+            .handle_request(
+                region_id,
+                RegionRequest::Drop(RegionDropRequest {
+                    fast_drop_path: false,
```

Comment:
I suggest introducing a force_drop_all_logical_tables flag. When set to true, it will drop the physical table along with all logical tables associated with it.
```suggestion
                    force_drop_all_logical_tables: false,
```